src/PreferencesDialog.ui

```diff
@@ -20,7 +20,7 @@
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
-      <number>0</number>
+      <number>4</number>
```

Comment:
The PR still seems to be changing this "currentIndex" number.  That's not correct is it?

Reply:
Good point, I did the UI modification via Qt Designer again and it looks like I made some unintended changes.